acceptance/install/install_test.go

```diff
@@ -0,0 +1,221 @@
+package install_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/puppetlabs/pdkgo/acceptance/testutils"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_PctInstall_InstallsTo_DefaultTemplatePath(t *testing.T) {
+	testutils.SkipAcceptanceTest(t)
+
+	// Setup
+	templatePath := getDefaultTemplatePath()
```

Comment:
I'm not keen on this function. having thought about it I believe we need to introduce a cli command which returns `viper.GetString("templatepath")`. This will help improve the UX and also be useful for testing.